test/Spec/ExampleSpec.hs

```diff
@@ -0,0 +1,11 @@
+module Spec.ExampleSpec where
```

Comment:
:information_source: Checked in a test. If there is a reason not to, we can drop it.